packages/nameguard-react/src/index.ts

```diff
@@ -7,8 +7,9 @@ export { SettingsModal } from "./components/SettingsModal/SettingsModal";
 
 export { ReportBadge } from "./components/ReportBadge";
 export { ReportIcon } from "./components/ReportIcon";
-export { ResultCodeIcon } from "./components/Report/ResultCodeIcon";
+export { CheckResultCodeIcon } from "./components/Report/CheckResultCodeIcon";
 export { RatingIcon, RatingIconSize } from "./components/Report/RatingIcon";
+export { NGWebsiteReportBadge } from "./components/Report/NGWebsiteReportBadge";
```

Comment:
This UI component should not be in the `nameguard-react` package. That package is for everyone who might want to build a React UI using NameGuard. This UI component is specific to the NameGuard website and therefore it should be moved.

Reply:
@lightwalker-eth `NGWebsiteReportBadge` is used both in https://nameguard.io/ and in https://nameguard.io/ modal. The first leaves inside `apps/website` while the second, is a component inside `@namehash/nameguard-react`. We cannot import components from `apps/website` to `@namehash/nameguard-react` and this is why I had to move `NGWebsiteReportBadge` to `@namehash/nameguard-react`. For next step, I'd recommend:

1. Providing this `NGWebsiteReportBadge` for importing from `@namehash/nameguard-react` as part of the package utilities OR;
2. Using `NGWebsiteReportBadge` in apps/website and using similar implementation but not same component in `@namehash/nameguard-react`'s `SearchEmptyState` component;

Let me know your thoughts around this 👍🏼

Reply:
@FrancoAguzzi Ah you’re right I forgot about that.

Let’s keep it inside ‘nameguard-react’. Also, given the important correction you shared, we should probably rename the component to something more generic. I only suggested a name that was specific for the NG website due to my misunderstanding.